bot.js

```diff
@@ -373,3 +377,26 @@ function writeToTimerFile(channel){
 	}
 }
 
+/** Adding below for command list helper **/
+//Executed: for any !zakum command.
+function listCommands(channel, isUserAdmin){
```

Comment:
Can we move the messages to a JSON object with an array of messages in the constants file? Then we can reduce this to ~2 lines of code by iterating over the messages and make it look a little cleaner 😄